app_config_sample

```diff
@@ -9,3 +9,6 @@ time_format:%H:%M:%S
 separator:
 ps2:➜
 fresh_file
+user_name:
+host_name:
+show_tag_name:true
```

Comment:
Everything is ok with the new keys:values. 
The only change needed here is that the last line should be the "fresh_file" string.

Reply:
Done !